docs/security/authorization/spark/build.md

```diff
@@ -53,14 +53,16 @@ Sometimes, it may be incompatible with other Spark distributions, then you may n
 For example,
 
 ```shell
-build/mvn clean package -pl :kyuubi-spark-authz_2.12 -DskipTests -Dspark.version=3.0.2
+build/mvn clean package -pl :kyuubi-spark-authz_2.12 -am -DskipTests -Pspark-3.2 -Dspark.version=3.2.1
 ```
 
 The available `spark.version`s are shown in the following table.
 
 |   Spark Version   | Supported |                                                         Remark                                                         |
 |:-----------------:|:---------:|:----------------------------------------------------------------------------------------------------------------------:|
 |      master       |     √     |                                                           -                                                            |
+|       3.5.x       |     √     |                                                           -                                                            |
+|       3.4.x       |     √     |                                                           -                                                            |
 |       3.3.x       |     √     |                                                           -                                                            |
 |       3.2.x       |     √     |                                                           -                                                            |
 |       3.1.x       |     √     |                                                           -                                                            |
```

Comment:
I think we also removed support for 3.1 and maybe 3.2

Reply:
Do you have a clue that which releases for dropping support of Spark 3.1 and 3.2?

Reply:
OK, we are dropping Spark 3.1 support since 1.10.0 on-go release, while Spark 3.2 is still supported and tested.